facade/src/main/java/org/jboss/pnc/facade/providers/GenericSettingProvider.java

```diff
@@ -117,7 +117,6 @@ public void setPNCVersion(String version) {
         GenericSetting pncVersion = createGenericParameterIfNotFound(PNC_VERSION);
         pncVersion.setValue(version);
         genericSettingRepository.save(pncVersion);
-        notifier.sendMessage(GenericSettingNotification.newAnnoucement(version));
```

Comment:
why removing this?

Reply:
This was an error from my previous PR which a noticed radnomly now and did not want to create a special PR for it.

Reply:
can you put it in separate commit at least then?

Reply:
Sure